### PR TITLE
Issue: 331 image host ideas

### PR DESCRIPTION
**Pull request by accordingtoherr**
I added in the Contributing.MD file some ideas that can solve Issue #331

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -78,3 +78,9 @@ After the code has been reviewed, the Team Member may suggest several things to
 ## Issue Triage
 
 The project Team Members may not have time to investigate every issue. If you find an older issue that nobody has commented on, it would be very appreciated if you could reproduce the issue, make a simple test case if necessary, and potentially suggest the root cause and/or solutions.
+
+
+## Image Hosting Guidelines
+Currently, this project utilizes the Tessel Project Imgur, older image assets are on AWS from Technical Machine days, and we have a few miscellaneously hosted images too.
```
Comment (Frijol) [inline CONTRIBUTING.md:84]: Markdown needs an extra newline in order to make this a paragraph break

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,3 +78,9 @@ After the code has been reviewed, the Team Member may suggest several things to
 ## Issue Triage
 
 The project Team Members may not have time to investigate every issue. If you find an older issue that nobody has commented on, it would be very appreciated if you could reproduce the issue, make a simple test case if necessary, and potentially suggest the root cause and/or solutions.
+
+
+## Image Hosting Guidelines
+Currently, this project utilizes the Tessel Project Imgur, older image assets are on AWS from Technical Machine days, and we have a few miscellaneously hosted images too. 
+Moving forward, our process will be to streamline image hosting through Tessel Project's Imgur. Older assets on AWS and elsewhere can be retroactivley moved over in the future. The Imgur should have folders per project/feature/whatever the image is of.
```
Comment (Frijol) [inline CONTRIBUTING.md:85]: Overall this is a good idea but it does create an access bottleneck; only a few people have keys to the Tessel Imgur account, and I'd love to decentralize more. Two other ideas: what if the images were hosted on a specific Tessel repo that people could PR? Or, what if images were hosted on anyone's imgur account (and thus just public links)? Curious about tradeoffs, e.g. the Github hosting would be annoying to clone down, but Github now has a pretty slick in-browser file upload so maybe that's a non-issue now?

**Comment by accordingtoherr** [comment]
i like the specific repo one - it makes it easy for anyone with access and does not really affect the main project

**Comment by Frijol** [comment]
Looks great! I just made a (very basic) repo if you want to change it to link to that: https://github.com/tessel/image-host

**Comment by accordingtoherr** [comment]
added link

**Review by Frijol — APPROVED** [review]

**Pull request merged**